ntbs-integration-tests/NotificationPages/ManualTestResultEditPagesTests.cs

```diff
@@ -210,10 +213,58 @@ public async Task PostDelete_ReturnsSuccessAndRemovesResult()
             var result = await SendPostFormWithData(editDocument, formData, editUrl, "Delete");
 
             // Assert
-            result.AssertRedirectTo(GetPathForId(NotificationSubPaths.EditTestResults, NotificationId));
+            result.AssertRedirectTo(GetPathForId(NotificationSubPaths.EditTestResults, notificationId));
             var testsListPage = await Client.GetAsync(GetRedirectLocation(result)); // Follow the redirect to see results table
             var testsListDocument = await GetDocumentAsync(testsListPage);
             Assert.Null(testsListDocument.GetElementById($"manual-test-result-{TEST_TO_DELETE_ID}"));
         }
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task GetFilteredSampleTypesForManualTestType_ReturnsExpectedValues(bool isEdit)
+        {
+            // Arrange
+            const int manualTestTypeId = (int)ManualTestTypeId.Histology;
+            const string handlerPath = "FilteredSampleTypesForManualTestType";
+            var endpointPath = $"{NotificationSubPath}{(isEdit ? "0/" : "")}{handlerPath}";
+            var endpointUrl = GetPathForId($"{endpointPath}?value={manualTestTypeId.ToString()}", 0);
```

Comment:
I think it's a worry that we're constructing a query string ourselves here, whilst `GetPathForId` is capable of it too. It's not a codepath we are hitting currently, but in theory we could end up with multiple `?` signs in the url.
Should we let `GetPathForId` accept an optional dictionary of query string values instead, and passing it through to a single place that does url construction?